### DISCUSSION TITLE
옵션 기능 구현

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#249] [5. Q&A]
![photo_2022-01-24_16-59-10](https://user-images.githubusercontent.com/13131384/150743756-68b934a5-0449-4b5e-a9b2-5c82e694d438.jpg)

설정 옵션 중에서 방 초대 받기, 이름 표시, 말 풍선 표시를 구현하고 싶습니다.

이름 표시를

ZepetoPlayers.instance.floatingUIData.showNickNameLocalPlayer
ZepetoPlayers.instance.floatingUIData.showNickNameOtherPlayers

위와 같이 접근 하여 변수를 바꿔도 이름이 그래로 보였습니다.

그래서 추측하기로 해당 데이터를 가지고 셋팅을 하는 것으로 보이는데 해당 부분만 따로 접근하여 변경하여 적용할 수 있는 함수 부분이 있을까요?

또한 방 초대 받기, 말 풍선 표시의 유무를 설정 할 수 있게 접근 할 수 있는 방법이 있는지 궁금합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.
Room 초대와 관련된 옵션설정은, ZEPETO World Setting 탭에서 설정할 수 있습니다.
![world_project_setting](https://user-images.githubusercontent.com/93751484/150902195-6688a77c-38af-4a9e-b52c-38bae97ed902.png)

또한, 말풍선 표시/닉네임 노출 여부는
ZepetoPlayers 컴포넌트에서 옵션으로 설정가능합니다. (해당 옵션의 적용은 앱에서 확인이 가능합니다. - 에디터에서는 확인 불가)
해당 가이드를 참고해주세요. 
[https://studio.zepeto.me/guides/character-control](https://studio.zepeto.me/guides/character-control)

도움이 되셨으면 좋겠습니다.
감사합니다.

**Comment by strangestraw** [comment]
런타임에 닉네임을 켜고 끌 수 있는 방법은 없나요?

**Reply by moondory77-naverz** [reply]
안녕하세요.
캐릭터 말풍선 표시 / 닉네임 노출 선택 옵션의 경우, 캐릭터를 인스턴스화 하는 시점에 적용되며
현재 런타임 중, 임의의 캐릭터에 대해 닉네임 노출여부를 on/off 하는 옵션은 따로 제공하고 있지 않습니다.

감사합니다.
